Firestore/core/src/remote/bloom_filter.h

```diff
@@ -113,6 +113,10 @@ class BloomFilter final {
 
 bool operator==(const BloomFilter& lhs, const BloomFilter& rhs);
 
+bool operator!=(const BloomFilter& lhs, const BloomFilter& rhs);
+
+bool CompareBits(const BloomFilter& lhs, const BloomFilter& rhs);
```

Comment:
Please remove CompareBits() from the header file since it's just an implementation detail. In fact, it should be put into an anonymous namespace in the .cc file to scope it to the file.